spacy/pipeline/entity_linker.py

```diff
@@ -204,22 +232,33 @@ def __init__(
         self.incl_prior = incl_prior
         self.incl_context = incl_context
         self.get_candidates = get_candidates
+        self.get_candidates_batch = get_candidates_batch
         self.cfg: Dict[str, Any] = {"overwrite": overwrite}
         self.distance = CosineDistance(normalize=False)
         # how many neighbour sentences to take into account
-        # create an empty KB by default. If you want to load a predefined one, specify it in 'initialize'.
+        # create an empty KB by default
         self.kb = empty_kb(entity_vector_length)(self.vocab)
         self.scorer = scorer
         self.use_gold_ents = use_gold_ents
+        self.candidates_batch_size = candidates_batch_size
         self.threshold = threshold
 
-    def set_kb(self, kb_loader: Callable[[Vocab], KnowledgeBase]):
+        if candidates_batch_size < 1:
+            raise ValueError(Errors.E1044)
+
+    def set_kb(
+        self,
+        kb_loader: Union[
+            Callable[[Vocab, Type[KnowledgeBase]], KnowledgeBase],
```

Comment:
What's this additional type for? (it's also not in the doc string)

Reply:
Leftover, removed in next commit.